app/src/components/modal/modal_transaction/index.tsx

```diff
@@ -137,7 +137,7 @@ export const ModalTransaction = (props: Props) => {
             ? 'Transaction Confirmed'
             : txState === TransactionStep.error
             ? 'Transaction Failed'
-            : ''}
+            : 'WHAAAAT'}
```

Comment:
I feel like we shouldn't keep this 😆

Reply:
ohh yeah sorry about that I was testing some shit...